libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -1176,6 +1176,7 @@ type FeatureStatusBasePut errs featureName =
     :> "features"
     :> KnownTeamFeatureNameSymbol featureName
     :> ReqBody '[Servant.JSON] (TeamFeatureStatus 'WithoutLockStatus featureName)
+    :> QueryParam "ttl" TeamFeatureTTLValue
```

Comment:
no time to check, but is this optional (with default "unlimited"?  if not please change it, to make sure ibis won't break over this change!

Reply:
Should the default be `unlimited` or 8 weeks?